### PR TITLE
[#1931] Improvement(server-common): remove local variables for JettyServerConfig

### PR DESCRIPTION
**Pull request by YxAc**
### What changes were proposed in this pull request?
- remove local variables for JettyServerConfig

### Why are the changes needed?
Fix: #1931 

### Does this PR introduce _any_ user-facing change?
N/A

### How was this patch tested?
NO

### CODE REVIEW

**Comment by YxAc** [comment]
@justinmclean @qqqttt123  Please take a look this PR when you are free. Thank you both.

File: server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java [hunk 1]
```diff
@@ -368,27 +368,23 @@ private JettyServerConfig(Map<String, String> configs) {
...
+      this.keyStorePath = null;
+      this.keyStorePassword = null;
+      this.managerPassword = null;
+    }
+    if (this.enableHttps && this.enableClientAuth) {
+      this.trustStorePath = internalConfig.get(SSL_TRUST_STORE_PATH);
+      this.trustStorePassword = internalConfig.get(SSL_TRUST_STORE_PASSWORD);
+    } else {
+      this.trustStorePath = null;
+      this.trustStorePassword = null;
```
Comment (justinmclean) [inline server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java:386]: Given the local variables no longer exist, there is probably no need to use "this." While we are there, we might remove unneeded "this." from other places in that method. Also, minThreads/maxThreads exist as local variables; you might consider renaming those.

**Review by justinmclean — CHANGES_REQUESTED** [review]

File: server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java (hunk 1 above)
Comment (qqqttt123) [reply server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java:386]: In the constructor method, we use the format `this.field`. It's code style of our project. If we decide to change this, maybe we will many places in other classes.

Comment (YxAc) [reply server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java:386]: @justinmclean  I think _minThreads/maxThreads_ may be reassigned and log print more information, which is why local variables are used.

Comment (justinmclean) [reply server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java:386]: Nothing to do with this PR @qqqttt123 but I don't see this. It is inconsistantly used (it is used when paramaters of the same name are passed into the constructor which is needed), it's not documented anywhere and we have no check for it.

Comment (qqqttt123) [reply server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java:386]: Emm ... Maybe it's inconsistent. But I follow the style and the code which I read also follows the style. Maybe it's optional.

Comment (YxAc) [reply server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java:386]: So i think both the style and min/maxThreads are ok.
Is there anything else I need to do for this PR?  thx.

Comment (justinmclean) [reply server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java:386]: Or @qqqttt123, even better and if possible, use Spotless to enforce this.

Comment (qqqttt123) [reply server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java:386]: > Or @qqqttt123, even better and if possible, use Spotless to enforce this.

It's an optional rule. Just for me, there are many places to follow it. I just feel that we don't need to change it.

Comment (justinmclean) [reply server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java:386]: Even if it is optional, it should be documented.

Comment (qqqttt123) [reply server-common/src/main/java/com/datastrato/gravitino/server/web/JettyServerConfig.java:386]: > Even if it is optional, it should be documented.

Make sense.

**Review by justinmclean — APPROVED** [review]
Could be improved a little further, but this fixes the issue.

**Pull request merged**